proposals/cel_finegrained_exceptions.md

```diff
@@ -0,0 +1,320 @@
+# Meta
+[meta]: #meta
+- Name: The CEL Fine-Grained Exceptions
+- Start Date: 2025-07-29
+- Author(s): [Mariam Fahmy](https://github.com/MariamFahmy98)
+
+# Table of Contents
+[table-of-contents]: #table-of-contents
+- [Overview](#overview)
+- [Motivation](#motivation)
+- [Proposal](#proposal)
+- [Implementation](#implementation)
+
+# Overview
+[overview]: #overview
+
+This proposal outlines the design for a fine-grained exception for Kyverno's CEL-based ValidatingPolicy.
+
+The goals of this proposal are to:
+
+1. **Excluding Specific Images:** A straightforward method to exempt resources from policy enforcement based on their container image names.
```

Comment:
```suggestion
1. **Excluding Specific Images:** A straightforward method to exempt resources from policy enforcement based on their container image reference.
```

Reply:
Would be good to explain why this is important.

Container names, resource names, etc. can be spoofed. An image reference cannot.